controllers/userController.js

```diff
@@ -0,0 +1,31 @@
+const { userService } = require('../services')
+const { catchAsync } = require('../utils/error')
+
+const signUp = catchAsync(async (req, res) => {  
+  const { name, email, residentNumberFront, residentNumberBack, password } = req.body;
+
+	const insertId = await userService.signUp(name, email, residentNumberFront, residentNumberBack, password);
+
+	res.status(201).json({ insertId });
+})
```

Comment:
indentation을 잘 맞춰주세요~

```suggestion
const signUp = catchAsync(async (req, res) => {  
  const { name, email, residentNumberFront, residentNumberBack, password } = req.body;

  const insertId = await userService.signUp(name, email, residentNumberFront, residentNumberBack, password);

  res.status(201).json({ insertId });
})
```